python/prophet/diagnostics.py

```diff
@@ -120,8 +122,11 @@ def map(self, func, *iterables):
     
     df = model.history.copy().reset_index(drop=True)
     horizon = pd.Timedelta(horizon)
-
     predict_columns = ['ds', 'yhat']
+    
+    if extra_output_columns is not None:
```

Comment:
Small one: could we move this after L130-131? To be sure that we're not creating duplicate columns.